_rules/SC1-2-1-audio-transcript.md

```diff
@@ -21,26 +18,28 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or with a "play button".
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button".  If a play button exists the rule applies if the button is [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
 
-A play button is an interactive element that when activated, plays the audio. The play button must be [visible on the page](#visible-on-the-page) or [included in the accessibility tree](#included-in-the-accessibility-tree).
+A play button is an interactive element that when activated, plays the audio. The play button must be [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
 
 ### Expectation
 
-The auditory information of each test target is available through a caption, or through a [text transcript](#text-transcript) that is available either on the page or through a link.
+The auditory information of each test target is available through a text transcript that is available either on the page or through a link.
+
+**Note**: A "text transcript" in the context of this rule is defined in WCAG 2 as a [alternative for time based media](https://www.w3.org/TR/WCAG21/#dfn-alternative-for-time-based-media).
 
 ## Assumptions
 
 *There are currently no assumptions*
 
 ## Accessibility support
 
-While the HTML specifications allows the use of `track` elements inside of `audio` elements, there is currently limited to no support for this in major web browsers.
+There are no major accessibility support issues known for this rule.
 
 ## Background
 
-- https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html
-- https://www.w3.org/TR/WCAG20-TECHS/G158.html
+- [Understanding SC 1.2.1:Audio-only and Video-only (Prerecorded)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html) 
+- [G158:Providing an alternative for time-based media for audio-only content](https://www.w3.org/TR/WCAG20-TECHS/G158.html)
 
 ## Test Cases
```

Comment:
Add a failed test case with controls and without a transcript (currently, all failed examples have incorrect transcripts)